RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptionsViewModel.swift

```diff
@@ -49,27 +49,34 @@ class ManageSubscriptionsViewModel: ObservableObject {
     @Published
     private(set) var refundRequestStatusMessage: String?
 
-    private var purchasesProvider: ManageSubscriptionsPurchaseType
+    private let purchasesProvider: ManageSubscriptionsPurchaseType
```

Comment:
Oopsies, I just realized this should be `ManageSubscriptionsPurchasesType`. I can fix it in another PR